addons/website/static/src/snippets/s_image_gallery/002.scss

```
@@ -0,0 +1,330 @@
+        img {
+            cursor: pointer;
+        }
+    }
+
+    &.o_slideshow {
+        --o-carousel-indicators-dots-size: .75rem;
+        --o-carousel-controllers-size: #{map-get($spacers, 5)};
+
+        .carousel {
+            &, .carousel-inner {
+                height: var(--snippet-preview-height, 100%);
+            }
+
+            .carousel-item.active, .carousel-item-next, .carousel-item-prev {
+                display: flex;
+                align-items: center;
+                height: var(--snippet-preview-height, 100%);
+            }
+
+            .media_iframe_video {
+                // 70% because the carousel controls are each 15% and the user
+                // must be able to click on the video buttons.
+                width: 70%;
+                height: 100%;
+            }
+        }
+
+        > .container, > .container-fluid, > .o_container_small {
+            height: var(--snippet-preview-height, 100%);
+        }
+
+        > .container-fluid {
+            padding: 0;
+        }
+
+        .o_carousel_controllers {
+            display: contents;
+        }
+
+        .carousel-indicators {
+            align-items: center;
+        }
+
+        .carousel-control-prev, .carousel-control-next {
+            border-radius: $border-radius;
+        }
+
+        .carousel-control-prev-icon, .carousel-control-next-icon {
+            width: var(--o-carousel-controllers-size);
+            height: var(--o-carousel-controllers-size);
+            background-size: 50%;
+        }
+
+        // Arrows - Default
+        &.s_image_gallery_arrows_default {
+            .carousel-control-prev, .carousel-control-next {
+                &:hover {
+                    background: rgba($carousel-dark-indicator-active-bg, .2);
+                }
+            }
+            .carousel-dark {
+                .carousel-control-prev, .carousel-control-next {
+                    &:hover {
+                        background: rgba($white, .2);
+                    }
+                }
+            }
+        }
+
+        // Arrows - Boxed / Rounded
+        &.s_image_gallery_arrows_boxed, &.s_image_gallery_arrows_rounded {
+            .carousel-control-prev-icon, .carousel-control-next-icon {
+                border-radius: $border-radius;
+                border: $border-width solid $border-color;
+                background-color: $carousel-dark-indicator-active-bg;
+            }
+        }
+
+        // Arrows - Rounded
+        &.s_image_gallery_arrows_rounded {
+            .carousel-control-prev-icon, .carousel-control-next-icon {
+                border-radius: $border-radius-pill;
+            }
+        }
+
+        // Arrows - Hidden
+        &.s_image_gallery_arrows_hidden {
+            .carousel-control-prev-icon, .carousel-control-next-icon {
+                display: none;
+            }
+        }
+
+        // Indicators - Bars
+        &.s_image_gallery_indicators_bars .carousel-indicators > * {
```

Consider avoiding * selectors unless absolutely necessary. It improves code readability and reduces the risk of side effects if the DOM changes in the future. While `carousel-indicators` is likely safe (as it should only contain buttons), it's still a good practice 😉

edit: other occurrences as well 
 
```suggestion
        &.s_image_gallery_indicators_bars .carousel-indicators > button {
```